{
  "PublicDateAtUSN": "0001-01-01T00:00:00Z",
  "CRD": "0001-01-01T00:00:00Z",
  "Candidate": "CVE-2025-43929",
  "PublicDate": "2025-04-20T03:15:00Z",
  "References": [
    "https://www.cve.org/CVERecord?id=CVE-2025-43929",
    "https://ghostwriter.kde.org/documentation/#links",
    "https://github.com/0xBenCantCode/CVE-2025-43929",
    "https://github.com/kovidgoyal/kitty/compare/v0.40.1...v0.41.0",
    "https://hitman.services/cve-2025-43929/"
  ],
  "Description": "open_actions.py in kitty before 0.41.0 does not ask for user confirmation before running a local executable file that may have been linked from an untrusted document (e.g., a document opened in KDE ghostwriter).",
  "UbuntuDescription": "",
  "Notes": null,
  "Bugs": [
    "http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=1103691"
  ],
  "Priority": "medium",
  "DiscoveredBy": "",
  "AssignedTo": "",
  "Patches": {
    "kitty": {
      "devel": {
        "Status": "needs-triage",
        "Note": ""
      },
      "esm-apps/focal": {
        "Status": "needs-triage",
        "Note": ""
      },
      "esm-apps/jammy": {
        "Status": "needs-triage",
        "Note": ""
      },
      "esm-apps/noble": {
        "Status": "needs-triage",
        "Note": ""
      },
      "focal": {
        "Status": "needs-triage",
        "Note": ""
      },
      "jammy": {
        "Status": "needs-triage",
        "Note": ""
      },
      "noble": {
        "Status": "needs-triage",
        "Note": ""
      },
      "oracular": {
        "Status": "needs-triage",
        "Note": ""
      },
      "plucky": {
        "Status": "needs-triage",
        "Note": ""
      },
      "upstream": {
        "Status": "released",
        "Note": "0.41.0"
      }
    }
  },
  "UpstreamLinks": {
    "kitty": [
      "https://github.com/kovidgoyal/kitty/commit/ce5cfdd9caf44c538af800a07162e1f49bd53c35"
    ]
  }
}